Program.cs

```diff
@@ -1,7 +1,13 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a number for a = ");
+         int a = int.Parse (Console.ReadLine ());
+         Console.Write ("Enter a number for b = ");
+         int b = int.Parse (Console.ReadLine ());
+         void Swap (int x, int y) => (a, b) = (b, a);

```

Comment:
Incorrect usage of method with unused parameter of x and y.
Method result is incorrect as if I use Swap(100, 100), a and b still gets swapped.
Read [ref](https://www.w3schools.blog/c-sharp-call-by-reference).